### ISSUE TITLE
What to do after installation !?

### ISSUE THREAD

**Issue by ikus060** [buildbot/buildbot_travis#73]
I'm new to builtdbot. Now, I'm trying builtbot_travis cause I've very interest to see how it's working. So far, I've install builbot on a new server (a VM). I've install docker and builtbot on it.

I have access to builtbot UI, but then what ?

I guess I need to create new worker or new project but I don't have any clue about the right way to do it. I've made various experimentation without luck.

**Comment by perlun** [comment]
@ikus060 Do you have any previous experience with CI and/or Travis?

**Comment by ikus060** [comment]
I do have experience with Travis. But not with Buildbot. I spent quite some time trying to get it working. I don't have the full details of what went wrong but it wasn't a nice experience. 
1. As I recall, the UI was not displaying properly. Kinda of a refresh issue or caching issues. I manage to get it working somehow. 
2. My Docker installation was configured with TLS. Seams buildbot is not supporting this.
3. Then I failed to properly configure the Docker image to be used for my project.

Basically, I waned a replacement for Travis, which is just "working" out of the box. buildbot_travis need more configuration to get it working. I would blame the lake of documentation for this.
